data/src/main/scala/io/prediction/data/api/EventServer.scala

```diff
@@ -61,33 +61,37 @@ class EventServiceActor(
       new DateTimeJson4sSupport.Serializer
   }
 
-  val log = Logging(context.system, this)
-
+  val logger = Logging(context.system, this)
   // we use the enclosing ActorContext's or ActorSystem's dispatcher for our
   // Futures
-  implicit def executionContext: ExecutionContext = actorRefFactory.dispatcher
+  implicit def executionContext: ExecutionContext = context.dispatcher
```

Comment:
`actorRefFactory.dispatcher` is a default actor system dispatcher while `context.dispatcher` is the dispatcher actor execution on, it can be different from the actor system default one.